handlers/new.go

```diff
@@ -11,10 +11,19 @@ import (
 	"net/http"
 	"path"
 	"strconv"
+	"time"
 
-	"github.com/coreos/go-etcd/etcd"
+	"github.com/coreos/etcd/client"
+	"golang.org/x/net/context"
 )
 
+var cfg = client.Config{
+	Endpoints: []string{"http://127.0.0.1:2379"},
```

Comment:
the goal of this pr was moreso to duplicate existing behavior with the new library. I'd like to do this in another PR.

... also because #42 depends on this.